be/src/util/arrow/row_batch.cpp

```diff
@@ -86,6 +86,11 @@ Status convert_to_arrow_type(const TypeDescriptor& type, std::shared_ptr<arrow::
     case TYPE_DECIMALV2:
         *result = std::make_shared<arrow::Decimal128Type>(27, 9);
         break;
+    case TYPE_DECIMAL32:
+    case TYPE_DECIMAL64:
+    case TYPE_DECIMAL128:
+        *result = std::make_shared<arrow::Decimal128Type>(type.precision, type.scale);
```

Comment:
`convert_to_starrocks_type` also need to cover `arrow::Decimal128Type`?

Reply:
convert_to_starrocks_type is only used in unittest